packages/loader/driver-utils/src/blobAggregationStorage.ts

```diff
@@ -102,7 +102,7 @@ export abstract class SnapshotExtractor {
     public async unpackSnapshotCore(snapshot: ISnapshotTree, level = 0): Promise<void> {
         // for now only working at data store level, i.e.
         // .app/DataStore/...
-        if (level >= 3) {
+        if (level > BlobAggregationStorage.dataStoreLevel) {
```

Comment:
@vladsud @arinwt, ".channels" in the summary tree can be disabled by setting `disableIsolatedChannels` to true. We should account for that for the data store level while packing / unpacking right?

Reply:
I'm not familiar with what exactly this code is doing, but for tracking the absolute depth level of a given datastore/dds within the summary tree, we do need to consider that .channels may or may not be there. If we are looking at an `ISnapshotTree`, we can identify if it will exist by looking at the .metadata blob (see code in ContainerRuntime for this).

For writing a summary tree, we use the `disableIsolatedChannels` property in ContainerRuntime to make that decision.

Reply:
We can make unpacking unconditional (i.e. no checks) - there is not that much perf saving - scanning full tree does not take that long. For writing, we should rely on disableIsolatedChannels or better - some const that tells us the depth of data stores (i.e. exported by same package that exports disableIsolatedChannels )